python/CHANGELOG.rst

```diff
@@ -14,6 +17,7 @@
 **Breaking changes**
 
 - The argument to ``ts.dump`` and ``tskit.load`` has been renamed `file` from `path`.
+- Arguments to ``Tree.newick()`` are now all positional.
```

Comment:
This is a bit gratuitous, I think. People will be using this code and it will probably break